components/src/core/components/InputField/input-group.scss

```diff
@@ -34,3 +34,11 @@
   margin-left: 2px;
   word-break: break-all;
 }
+
+.oxd-input-group__label-icon {
+  width: $oxd-input-group-label-icon-size;
+  height: $oxd-input-group-label-icon-size;
```

Comment:
Force the label icon to be the same size, regardless of icon type (svg / icon font).